database/mongo/seed.js

```diff
@@ -0,0 +1,65 @@
+const faker = require('faker/locale/en_US');
+const Jabber = require('jabber');
+const moment = require('moment');
+const Review = require('./index.js');
+// const Review = require('./mongoSchema');
+
+// moment(randomDate).fromNow()
+
+const buzzWords = ['great food', 'eat', 'service', 'drinks', 'loved', 'never again', 'dessert', 'main menu', 'appetizers', 'enjoyed', 'hated', 'server', 'noise', 'waitress', 'waiter', 'slow', 'fast', 'go again', 'clean', 'dirty', 'kids', 'our group', 'slow service', 'cold food', 'awesome drinks', 'fun time', 'romantic atmosphere', 'dinner', 'lunch', 'breakfast', 'hot plate', 'smelled wonderful', 'no alcohol', 'bartender', 'wine list', 'outdoor seating', 'large room', 'no tips', 'valet parking', 'fine dining', 'dishes', 'shrimp', 'steak', 'chicken', 'beef', 'pasta', 'cake', 'pastries', 'creme brulee', 'simmering', 'coffee', 'fresh food', 'candlelit', 'anniversary', 'birthday', 'party', 'with friends', 'evening', 'stuffed', 'would go again', 'recommend this place', 'restrooms clean', 'host greeting', 'lovely place', 'best view', 'indoor seating', 'waited an hour', 'risotto', 'baked salmon', 'fried rice', 'shrimp scampi', 'sushi', 'wait time'];
+
+const tags = ['Good for groups', 'Desserts', 'Appetizers', 'Drinks', 'Kid friendly'];
+
+const circleColors = ['#6c8ae4', '#d86441', '#bb6acd', '#df4e96'];
+
+const jabber = new Jabber(buzzWords, 2);
+
```

Comment:
Wow! This is so cool in terms of data generation.

Reply:
For sure.  John went all out with this lol

Reply:
Sorry to sneak in :) there are new features to generate names/email addresses as well in Jabber. Thanks for using the package. 
![image](https://user-images.githubusercontent.com/1720245/103010470-b3aee200-4538-11eb-9305-caba07f511d8.png)